source/test/Test.hh

```diff
@@ -16,6 +22,10 @@ struct TestHeader {
 struct TestData {
     EGG::Vector3f pos;
     EGG::Quatf fullRot;
+    // Added in maj0 min2
+    EGG::Vector3f extVel;
+    // Added in maj0 min3
+    EGG::Vector3f intVel;
```

Comment:
I think we can just say `// Added in 0.2` or `// Added in 1.3` (as an example) instead. I think the meaning is easily derivable without `maj` or `min`. Maybe we also put these on the same line as the declarations to limit line count? If clang-format even allows that. Just my preference I guess

Reply:
Resolved in commit c59fb22.